core-web/libs/contenttype-fields/src/lib/fields/binary-field/components/dot-binary-field-preview/dot-binary-field-preview.component.scss

```diff
@@ -0,0 +1,146 @@
+@use "variables" as *;
+
+:host {
+    display: block;
+    width: 100%;
+    height: 100%;
+
+    &::ng-deep {
+        .p-button-text .pi.p-button-icon {
+            color: $black;
+        }
+    }
```

Comment:
@fmontes, I noticed that some icons are black while others are `primary`. It might be a good idea to establish a rule for consistency. If necessary, we could create another class for black icons. I'm waiting for Melissa to discuss it.

Reply:
Done!